extensions/security-webauthn/runtime/src/main/java/io/quarkus/security/webauthn/WebAuthnRecorder.java

```diff
@@ -0,0 +1,77 @@
+package io.quarkus.security.webauthn;
+
+import java.security.SecureRandom;
+import java.util.Base64;
+import java.util.function.Supplier;
+
+import org.jboss.logging.Logger;
+
+import io.quarkus.arc.runtime.BeanContainer;
+import io.quarkus.runtime.RuntimeValue;
+import io.quarkus.runtime.annotations.Recorder;
+import io.quarkus.security.identity.IdentityProviderManager;
+import io.quarkus.vertx.http.runtime.HttpConfiguration;
+import io.quarkus.vertx.http.runtime.security.PersistentLoginManager;
+import io.vertx.ext.web.Router;
+import io.vertx.ext.web.handler.BodyHandler;
+
+@Recorder
+public class WebAuthnRecorder {
+
+    private static final Logger log = Logger.getLogger(WebAuthnRecorder.class);
+
+    final RuntimeValue<HttpConfiguration> httpConfiguration;
+    final RuntimeValue<WebAuthnRunTimeConfig> config;
+
+    //the temp encryption key, persistent across dev mode restarts
```

Comment:
@FroMage Is this key intended to be used in dev mode only ? If yes, then I think the right way to handle it to have it generated in the `deployment` as the `DevServices for WebAuthn` and then set as a runtime configuration property

Reply:
I don't think so, this is a direct copy of how it's done for Basic FORM auth.